Formula/glew.rb

```diff
@@ -13,10 +13,22 @@ class Glew < Formula
     sha256 "2b72bd7d59343ae64eaa87fd69f806759ac356a77300bb6b6a6ab40247384dc2" => :mavericks
   end
 
+  option "with-cmake", "Build GLEW with CMake"
+
+  depends_on "cmake" => [:build, :optional] if build.with? "cmake"
+
   def install
-    inreplace "glew.pc.in", "Requires: @requireslib@", ""
-    system "make", "GLEW_PREFIX=#{prefix}", "GLEW_DEST=#{prefix}", "all"
-    system "make", "GLEW_PREFIX=#{prefix}", "GLEW_DEST=#{prefix}", "install.all"
+    if build.with? "cmake"
+      cd "build" do
+        system "cmake", "./cmake", *std_cmake_args
+        system "make"
+        system "make", "install"
+      end
+    else
+      inreplace "glew.pc.in", "Requires: @requireslib@", ""
```

Comment:
This fix, which originated [here](https://github.com/Homebrew/legacy-homebrew/issues/22025#issuecomment-23732918), is not dependent on whether CMake is used. It would be good to get it reported upstream at some point so we can stop doing it, but until that happens, the same issue remains in the pkg-config file even when CMake is used.